STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,58 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+NOTE: trademark committee section will be moved to a distinct md file before merge.
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
```

Comment:
For avoidance of doubt, let's add ", membership" so that this reads "Only the Trademark Committee can redefine its own charter, membership, and authority ..."

Reply:
Done.